be/src/olap/task/engine_publish_version_task.cpp

```diff
@@ -60,7 +60,7 @@ OLAPStatus EnginePublishVersionTask::finish() {
             OLAPStatus publish_status = OLAP_SUCCESS;
             TabletInfo tablet_info = tablet_rs.first;
             RowsetSharedPtr rowset = tablet_rs.second;
-            LOG(INFO) << "begin to publish version on tablet. "
+            VLOG(3) << "begin to publish version on tablet. "
```

Comment:
I think we shouldn't remove these logs.
These logs are very important for track transaction issue.

Reply:
There are too many this kind of logs in our env. Why you need a "begin" log?

Reply:
Would better keep all publish version and transaction log. Our publish version and transaction performance is still poor. We need these logs to track issue.

Reply:
OK, I will leave the begin and finish log